src/App.tsx

```diff
@@ -1,61 +1,38 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodosWithUsers } from './type.ts/TodosWithUser';
+import { TodoForm } from './components/TodoForm';
+import { getUserById } from './utils/userId';
+import { useState } from 'react';
+
+const todos: TodosWithUsers[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [todosArr, setTodosArr] = useState(todos);
+
+
+  const addTodo = (newTodo: TodosWithUsers) => {
+    const maxId = todos.length > 0 ? Math.max(...todosArr.map(todo => todo.id)) : 0;
```

Comment:
The `maxId` calculation should use `todosArr` instead of `todos` to ensure it reflects the current state of the todos list. This will prevent any discrepancies in ID assignment if `todosArr` is modified.